### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by jsflax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,29 +3,105 @@
...
+
+let dog = Dog()
+dog.name = "Max"
+
+// Create a dog in the realm.
+try! realm.write {
+    realm.add(dog)
+}
+
+//  Set up the listener & Observe object notifications.
```
Comment (cbush) [inline README.md:58]:
```suggestion
//  Set up the listener & observe object notifications.
```

File: README.md [hunk 2]
```diff
@@ -3,29 +3,105 @@
...
+token = dog.observe { change in
+    switch change {
+    case .change(let properties):
+        for property in properties {
+            print("Property '(property.name)' changed to '(property.newValue!)'");
```
Comment (cbush) [inline README.md:63]: Did the \ get stripped? Or is it not needed?
```suggestion
            print("Property '\(property.name)' changed to '\(property.newValue!)'");
```

File: README.md [hunk 3]
```diff
@@ -3,29 +3,105 @@
...
-Please see the detailed instructions in our docs to add [Realm Objective-C](https://realm.io/docs/objc/latest/#installation) _or_ [Realm Swift](https://realm.io/docs/swift/latest/#installation) to your Xcode project.
+Please see the detailed instructions in our docs to add [Realm](https://docs.mongodb.com/realm/sdk/ios/install/) to your Xcode project.
 
 ## Documentation
 
-### Realm Objective-C
-
-The documentation can be found at [realm.io/docs/objc/latest](https://realm.io/docs/objc/latest).  
+The documentation can be found at [docs.mongodb.com/realm/sdk/ios/](https://docs.mongodb.com/realm/sdk/ios/).  
 The API reference is located at [realm.io/docs/objc/latest/api/](https://realm.io/docs/objc/latest/api/).
```
Comment (cbush) [inline README.md:103]: API reference is at https://docs.mongodb.com/realm-sdks/swift/latest/

**Review by cbush — COMMENTED** [review]
Nice

File: README.md (hunk 2 above)
Comment (jsflax) [reply README.md:63]: Think it's wrong on `realm.io`, where I copy and pasted from.

File: README.md [hunk 4]
```diff
@@ -3,29 +3,105 @@
...
+
+// Get the default Realm
+let realm = try! Realm()
+// Persist your data easily with a write transaction 
+try! realm.write {
+    realm.add(dog)
+}
+```
+## Live Objects: Build Reactive Apps
+Realm’s live objects mean data updated anywhere is automatically updated everywhere. If you need to freeze elements of your data for Reactive frameworks, you can do that, too.
```
Comment (ianpward) [inline README.md:43]: do we want to add an example of freezing and integrating into a reactive framework?

Comment (jsflax) [reply README.md:43]: I should probably just remove that line and show a SwiftUI example.

File: README.md [hunk 5]
```diff
@@ -3,29 +3,105 @@
...
+// Use the Realm as normal
+let dogs = realm.objects(Dog.self).filter("name contains 'Fido'")
+```
+## Data Sync
+The [MongoDB Realm Sync](https://www.mongodb.com/realm/mobile/sync) service makes it simple to keep data in sync across users, devices, and your backend in real-time.
 
 ## Getting Started
 
-Please see the detailed instructions in our docs to add [Realm Objective-C](https://realm.io/docs/objc/latest/#installation) _or_ [Realm Swift](https://realm.io/docs/swift/latest/#installation) to your Xcode project.
+Please see the detailed instructions in our docs to add [Realm](https://docs.mongodb.com/realm/sdk/ios/install/) to your Xcode project.
```
Comment (tgoyne) [inline README.md:98]: Maybe something like "We support installing Realm via Swift Package Manager, CocoaPods, Carthage, or by importing a dynamic XCFramework. For more information, see the detailed instructions in our docs."

**Review by tgoyne — COMMENTED** [review]
Build instructions still say that Xcode 11+ is required.

**Comment by ianpward** [comment]
It might be good to also include this non-sync tutorial as well - 
https://docs.mongodb.com/realm/tutorial/ios-swift/#std-label-ios-swift-tutorial-local

Also, is the link to the survey up here?

**Comment by jsflax** [comment]
@ianpward I added the link to the survey at the bottom. I can move it higher up if you'd like.

**Comment by ianpward** [comment]
Yeah lets move it to the top if possible - we will be removing it in a month or so

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,10 @@ x.y.z Release notes (yyyy-MM-dd)
 * None.
 
 ### Fixed
+* `RealmProperty<T?>` would crash when decoding a `null` json value.
+  ([Cocoa #7323](https://github.com/realm/realm-cocoa/issues/7323), since v10.8.0)
+* `@Persisted<T?>` would crash when decoding a `null` value.
+  ([#7332](https://github.com/realm/realm-cocoa/issues/7332), since v10.10.0).
```
Comment (tgoyne) [inline CHANGELOG.md:10]: Bad merge

Comment (jsflax) [reply CHANGELOG.md]: Fixed.

**Review by tgoyne — APPROVED** [review]

**Pull request merged**